caffe2/python/net_printer.py

```diff
@@ -185,8 +185,8 @@ def __str__(self):
 
 class Printer(Visitor, Text):
     def __init__(self, factor_prefixes=False, c2_syntax=True):
-        super(Visitor, self).__init__()
-        super(Text, self).__init__()
+        super().__init__()
+        super().__init__()
```

Comment:
I don't think these are equivalent. In general, need to be careful with multiple inheritance.